atd-etl/cris_import/cris_import.py

```diff
@@ -197,6 +214,63 @@ def get_secrets():
     return onepasswordconnectsdk.load_dict(client, REQUIRED_SECRETS)
 
 
+def mess_with_incoming_records_to_ensure_updates(map_state):
```

Comment:
This function is what ensures that every incoming record from CRIS is messed with a little to ensure that it's an update.

Reply:
could you add this as a docstring or some comments on the function?

Reply:
I did add a comment with usage instructions in the file containing this function now -- thank you for that suggestion!